### PR TITLE
Remove hard coded module version in java code

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: http-native/src/main/java/org/ballerinalang/net/http/HttpPackageUtil.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (BuddhiWathsala) [inline http-native/src/main/java/org/ballerinalang/net/http/HttpPackageUtil.java:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

File: http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java [hunk 2]
```diff
@@ -1788,6 +1791,11 @@ public static String encodeString(String value) {
         }
     }
 
+    static StrandMetadata getStrandMetadata(String method) {
+        return new StrandMetadata(BALLERINA_BUILTIN_PKG_PREFIX, PROTOCOL_HTTP, HttpPackageUtil.getHttpPackageVersion(),
```
Comment (warunalakshitha) [inline http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java:1795]: Can we make these in init function similar to io package

**Review by warunalakshitha — CHANGES_REQUESTED** [review]

File: http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java (hunk 2 above)
Comment (warunalakshitha) [reply http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java:1795]: This will create strand meta data for each request

Comment (chamil321) [reply http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java:1795]: Which init method?? `HttpPackageUtil.getHttpPackageVersion()`??

Comment (chamil321) [reply http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java]: Done

**Comment by warunalakshitha** [comment]
LGTM

**Review by warunalakshitha — APPROVED** [review]

**Pull request merged**